mono/mini/gshared.cs

```diff
@@ -1350,6 +1350,8 @@ class Class6 : IFace6 {
 
 	// FIXME: The runtime-invoke wrapper used by BeginInvoke is not found
 	[Category ("!FULLAOT")]
+	// FIXME: Wasm is single threaded and can't handle blocking waits
+	[Category ("!WASM")]
```

Comment:
Another time/never: Should be a characteristic, not a specific platform: `[Category ("Threads")]` or Multithreaded or  HasBlockingWaits, etc.